docs/src/stream.rst

```diff
@@ -133,6 +133,19 @@ API
     .. note::
         `server` and `client` must be handles running on the same loop.
 
+.. c:function:: int uv_reject(uv_stream_t* server)
+
+    This call is used in conjunction with :c:func:`uv_listen` to accept and
```

Comment:
I feel like we need to expand here. Please explain why someone would want to use this.